go.mod

```
@@ -11,14 +11,16 @@ require (
 	github.com/cenkalti/backoff/v4 v4.2.1
 	github.com/docker/go-units v0.5.0
 	github.com/dolmen-go/contextio v0.0.0-20200217195037-68fc5150bcd5
+	github.com/elastic/beats/v7 v7.17.18
 	github.com/elastic/e2e-testing v1.1.0
-	github.com/elastic/elastic-agent-autodiscover v0.6.8
+	github.com/elastic/elastic-agent-autodiscover v0.6.9
```

```suggestion
	github.com/elastic/elastic-agent-autodiscover v0.6.10
```

There's a newer version now updating the protobuf version to avoid a CVE.

---

v0.6.11 on the way that included v0.6.10

---

updated to v0.6.12